db/migrate/20161215230219_add_invoice_text_and_invoice_logo_to_enterprises.rb

```diff
@@ -0,0 +1,6 @@
+class AddInvoiceTextAndInvoiceLogoToEnterprises < ActiveRecord::Migration
+  def change
+    add_column :enterprises, :invoice_text, :text
+    add_column :enterprises, :invoice_logo, :boolean, default: false
```

Comment:
Nice idea! As I probably am the only person having applied this migration I changed it directly :)